### PR TITLE
Navigation

### PR DESCRIPTION
**Pull request by andrewbengordon**

### CODE REVIEW

**Comment by andrewbengordon** [comment]
Before you approve this, check it out and see if there are any weird visual bugs on large and small viewports

File: package-lock.json
Comment (garv5014) [inline package-lock.json:2]: Looks like an extra lock file got generated.

Comment (andrewbengordon) [reply package-lock.json]: I must have tried to npm i outside of the ui project

**Comment by Cwighty** [comment]
> Before you approve this, check it out and see if there are any weird visual bugs on large and small viewports

I don't notice anything

**Comment by garv5014** [comment]
> > Before you approve this, check it out and see if there are any weird visual bugs on large and small viewports
> 
> I don't notice anything

does this wrap up the food search story too or just navigation?

**Comment by Cwighty** [comment]
> > > Before you approve this, check it out and see if there are any weird visual bugs on large and small viewports
> > 
> > 
> > I don't notice anything
> 
> does this wrap up the food search story too or just navigation?

He just merged my branch into his, but mine isn't done yet

**Review by Cwighty — APPROVED** [review]
Your stuff looks good, my stuff isn't done but it's probably fine to merge for now

**Pull request merged**
